src/logger.js

```diff
@@ -12,10 +12,15 @@ var levels = [
     'assert'
 ];
 
+var BYTES_TO_MB = 1024 * 1024;
+
 /**
  * Creates a custom logger for this package
  */
 module.exports = {
+    bytesToMb: function(num) {
```

Comment:
I think this would be better in the common-test file as it is more about formatting test results